handsontable/src/plugins/multiColumnSorting/multiColumnSorting.scss

```diff
@@ -1,3 +1,11 @@
+.handsontable[dir="ltr"] div.htRight span[class*="sort-"] {
+  margin-right: 20px;
```

Comment:
It would be better to use CSS logical properties

Reply:
I don't think it would work here either. In case of `ltr` we need to react to `htRight` (and set the right margin) and in case of `rtl` we need to react to `htLeft` (and set the left margin).